src/elements/helpers.js

```diff
@@ -0,0 +1,9 @@
+export const createCustomElement = (tag, typeExtension) => {
+    if (window.WebComponents) {
+        const elem = document.createElement(tag);
+        elem.setAttribute('is', typeExtension);
+        return elem;
+    }
```

Comment:
In Chrome 70 `window.WebComponents` is defined (`{"flags":{"log":{}}}`), but it works without this block - need to find a valid test or maybe this block is redundant.

Reply:
The `setAttribute` method was required for it to work for me in modern Firefox.

Reply:
`document.createElement` method works fine in:
- Chrome 79
- Chrome 100
- Firefox 72
- Firefox 99
- webOS 1.2 emulator
- webOS 4 emulator

Dunno what about Safari